JsonDEmo/src/main/java/Demo.java

```diff
@@ -0,0 +1,21 @@
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+@JsonIgnoreProperties(ignoreUnknown = true)
+
+public class Demo {
+    public static  void main (String[] args) throws IOException {
+/*
+        ObjectMapper mapper = new ObjectMapper();
+        Person person = mapper.readValue(new File("persons.json"), Person.class);
+        System.out.println(person);
+*/
+        String url = "http://api.openweathermap.org/data/2.5/weather?q=London,uk&APPID=0e2441af1b772fa31363023705b6351c";
```

Comment:
Think of making the code an application: reusable, testable and shippable so others could start it and specify a city other than London without modifying the code and recompiling it. Hint: think which classes could be in the application, which fields/properties they have? You should not probably disclose the url string since it contains APPID, which is you personal secret information.